usl_pipeline/study_area_uploader/study_area_uploader/transformers/study_area_transformers.py

```diff
@@ -80,6 +96,8 @@ def prepare_and_upload_study_area_files(
     soil_class_mask_feature_property: Optional[str],
     work_dir: pathlib.Path,
     study_area_bucket: storage.Bucket,
+    input_non_green_area_soil_classes: set[int] = set(),
+    output_non_green_area_soil_class: int = DEFAULT_NON_GREEN_AREA_SOIL_CLASS,
```

Comment:
Why would a caller want to pass a non-default value here? Does it make sense to remove this as an argument?
Would they need to pass that same non-default value into `prepare_and_upload_citycat_input_files` as well?

Reply:
They actually don't want to pass a non-default value here. I removed this argument and the corresponding one in the citycat preparation function.